ocaml/network/networkd.ml

```diff
@@ -70,7 +70,20 @@ let handle_shutdown () =
 	Sys.set_signal Sys.sigint (Sys.Signal_handle stop);
 	Sys.set_signal Sys.sigpipe Sys.Signal_ignore
 
+(* Read the xcp-networkd.conf *)
+let read_config () =
+	let configargs = [
+		"enic-workaround-until-version", Config.Set_string Network_server.enic_workaround_until_version;
```

Comment:
On cream-bugfix there is a difference while passing config.
On trunk it is enic-workaround-until-version="x.x.x.x" and
on cream-bugfix its enic-workaround-until-version=x.x.x.x